### PR TITLE
Fix scale & neo4j

### PR DESCRIPTION
**Pull request by noboruma**
Tweak numbers, add caching to address scalability high load
Optimize neo4j queries to make them more robust (add timeout, add more limits)

### CODE REVIEW

File: deepfence_worker/cronjobs/threat.go [hunk 1]
```diff
@@ -2,14 +2,17 @@ package cronjobs
...
 
 	"github.com/ThreeDotsLabs/watermill/message"
 
 	"github.com/deepfence/golang_deepfence_sdk/utils/directory"
+	"github.com/deepfence/golang_deepfence_sdk/utils/log"
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 )
 
 var threatGraphRunning atomic.Bool
+var exploitabilityRunning atomic.Bool
```
Comment (gnmahanth) [inline deepfence_worker/cronjobs/threat.go:15]: I think these flags needs to be set and checked from redis as the jobs might be running on different workers

Comment (noboruma) [reply deepfence_worker/cronjobs/threat.go:15]: Good point, more globally I am not sure worker is actually a scalable process
For one, it has all the cron jobs, which should all be unique

**Comment by noboruma** [comment]
Merged

File: deepfence_worker/cronjobs/threat.go (hunk 1 above)
Comment (ramanan-ravi) [reply deepfence_worker/cronjobs/threat.go:15]: Scheduler is unique and triggers cron jobs. Worker should be scalable. @noboruma

**Pull request closed without merging**